test/parallel/test-http-malformed-request.js

```diff
@@ -25,11 +25,11 @@ const assert = require('assert');
 const net = require('net');
 const http = require('http');
 const url = require('url');
+const Countdown = require('../common/countdown');
 
 // Make sure no exceptions are thrown when receiving malformed HTTP
 // requests.
-
-let nrequests_completed = 0;
+const nrequests_completed = 0;
 const nrequests_expected = 1;
 
 const server = http.createServer(function(req, res) {
```

Comment:
If you use `common.mustCall` around the function callback here, it will do the same thing as the Countdown but better. Then you can just get rid of all the countdown stuff and make the `server.close()` unconditional.